mrcnn/model.py

```diff
@@ -999,8 +1000,10 @@ def build_fpn_mask_graph(rois, feature_maps, image_meta,
                            name='mrcnn_mask_bn4')(x, training=train_bn)
     x = KL.Activation('relu')(x)
 
-    x = KL.TimeDistributed(KL.Conv2DTranspose(256, (2, 2), strides=2, activation="relu"),
-                           name="mrcnn_mask_deconv")(x)
+    for i in range(num_deconv_layers):
+        x = KL.TimeDistributed(KL.Conv2DTranspose(256, (2, 2), strides=2, activation="relu"),
+                           name="mrcnn_mask_deconv" + str(i+1))(x)
```

Comment:
The first transposed conv should have the same name as before. In other words, `mrcnn_mask_deconv` rather than `mrcnn_mask_deconv1`. Otherwise, it will prevent loading the pre-trained COCO weights.

Reply:
I've change the first deconv layer's name to "mrcnn_mask_deconv".